src/pip/_internal/network/xmlrpc.py

```diff
@@ -1,5 +1,8 @@
 """xmlrpclib.Transport implementation
 """
+# The following comment should be removed at some point in the future.
```

Comment:
@pradyunsg @xavfernandez This one fixes CI failure introduced here https://github.com/pypa/pip/pull/7095

Reply:
Also, all opened PRs should be rebased on master before merge now, or this kind of CI failure would happen again.